### PR TITLE
Feature/tag management

### LINKED ISSUE #100: Adminille tagien hallintasivu
related to #18 

admin voi lisätä ja poistaa tageja

### LINKED ISSUE #102: backend logiikka tageille
related to #18 

db:ssä on jo tag taulu, backend logiikka yhdistettävä tietokantaan

### PR DESCRIPTION
**Pull request by miahro**
resolves #102 
resolves #100 

Added tag management functionality for admin

For student time logs backend and db already supports tags, but tags not implemented to student front. These are user story #18 and task #101 , which are still open.

### CODE REVIEW

File: backend/controllers/tags.js
Comment (Regularmute) [inline backend/controllers/tags.js:1]: Row 32, is the console.log('fetching tags') meant to go to production?

**Review by Regularmute — COMMENTED** [review]
I added some comments, otherwise things look pretty good for me.

File: frontend/src/components/TagManagementPage/TagsDashboard.js
[the remark this answers is not in the document]
Comment (miahro) [reply frontend/src/components/TagManagementPage/TagsDashboard.js:1]: Well spotted, unnecessary residues from development phase. Removed in latest commit

File: backend/controllers/tags.js
Comment (miahro) [reply backend/controllers/tags.js:1]: No, not meant for production, and removed in latest commit

**Review by Regularmute — APPROVED** [review]
Looks good to me

**Pull request merged**